Content/ReleaseBriefings/4_6/Section_BackendUpdates.tex

```diff
@@ -0,0 +1,38 @@
+%==========================================================================
+
+\begin{frame}[fragile]
+
+  {\Huge Backend Updates}
+
+  \vspace{10pt}
+
+\end{frame}
+
+
+%==========================================================================
+
+% Examples
+
+% note: always keep the [fragile] for your frames!
+
+\begin{frame}[fragile]{CUDA, SYCL and Serial}
+  \begin{itemize}
+      \item CUDA: Improved performance for \texttt{Kokkos::parallel\_reduce} on H100 and newer by increasing launch bounds
+      \item SYCL: Improved sorting performance for non-contiguous views with \texttt{RandomAccessIterator}
+      \item Serial: Reduce fences when using \texttt{Kokkos\_ENABLE\_ATOMICS\_BYPASS}
+  \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{HIP}
+  \begin{itemize}
+      \item Change block size deduction to prefer smaller blocks/teams if possible
+      \item Allocate memory with stream ordered semantics (\emph{i.e.}\ use \texttt{hipMallocAsync}) as default
```

Comment:
```suggestion
      \item Allocate memory with stream ordered semantics (\emph{i.e.}\ use \texttt{hipMallocAsync})
```
The option controlling that is IMPL_ only so as far as users are concerned we do `hipMallocAsync`